app/routes.py

```diff
@@ -1,2 +1,50 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.color = color
+
+    def to_planet_dict(self):
+        return dict(
+            id = self.id,
+            name = self.name,
+            description = self.description,
+            color = self.color
+        )
+
+planets = [
+    Planet(1, "Pluto", "Small", "Blue"),
+    Planet(2, "Mercury", "Hot, probably", "Red"),
+    Planet(3, "Mars", "Medium", "Orange")
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
```

Comment:
The Learn modules show naming the blueprint with a model prefix. This can help us distringuish among multiple blueprints if we have blueprints for multiple models defined in the same routes file.

But typically, we'll end up putting the blueprint for one set set of releated endpoints in each file, which allows us to name the _single_ bluepoint in that file simply `bp`, as was done in the Flasky live code.